packages/cli-kit/src/ui.ts

```diff
@@ -3,13 +3,40 @@ import {Input} from './ui/input'
 import {Select} from './ui/select'
 import {Bug, AbortSilent} from './error'
 import {remove, exists} from './file'
-import {info, content, token} from './output'
+import {info, content, token, logToFile} from './output'
 import {relative} from './path'
 import {isTerminalInteractive} from './environment/local'
 import {isTruthy} from './environment/utilities'
 import inquirer from 'inquirer'
+import {Listr as OriginalListr, ListrTask, ListrEvent, ListrTaskState} from 'listr2'
+
+export function newListr(tasks: ListrTask[], options?: object) {
```

Comment:
What about renaming this to `list` and call `listr.run()` at the end? That way, we can follow the same style as other components like `ui.prompt` and use it with `await ui.list()` instead of `ui.newListr(); await ui.run()`

Reply:
This is interesting. In light of your feedback, I looked at all the examples, and we do seem to always call `run` right after constructing the Listr. It would certainly keep it simpler to have much of the type info contained within `ui`.

Still, I think it's a change that should be its own PR, as the goal isn't behavioral change but rather to have a cleaner interface/boundary between `ui` and its dependents.